lib/semantics/scope.h

```diff
@@ -114,6 +125,16 @@ class Scope {
   // that are referenced by SourceName objects.
   void set_chars(std::string &&chars) { chars_ = std::move(chars); }
 
+  ImportKind importKind() const;
+  // Names appearing in IMPORT statements in this scope
+  std::set<SourceName> importNames() const { return importNames_; }
+
+  // Set the kind of imports from host into this scope.
+  // Return an error message for incompatible kinds.
+  std::optional<parser::MessageFixedText> set_importKind(ImportKind);
```

Comment:
Does this follow our naming convention?

Reply:
Yes, at least for a normal setter it's `set_` and the data member name.

This one isn't quite normal though, because it can return an error. So maybe it should be `SetImportKind` to indicate it doesn't follow the standard pattern. Either way is okay with me.